packages/webapp/src/components/ui/OfflineTableNotice/index.tsx

```diff
@@ -0,0 +1,53 @@
+/*!
+ * Copyright (c) Microsoft. All rights reserved.
+ * Licensed under the MIT license. See LICENSE file in the project.
+ */
+import { wrap } from '~utils/appinsights'
+import { useOffline } from '~hooks/useOffline'
+import { useTranslation } from '~hooks/useTranslation'
+import styles from './index.module.scss'
+import cx from 'classnames'
+
+export const OfflineTableNotice = wrap(function OfflineTableNotice() {
```

Comment:
I wonder if this component should have a different name, as it shows _no results_ message as well?